client/src/components/HomiesGallery.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import MenuCard from './MenuCard';
+import YogaMenu from '../assets/hobbies/yoga.jpg';
+import Museum from '../assets/hobbies/museum.jpeg';
+import Lisa from '../assets/profile/Lisa.jpeg';
+import Tim from '../assets/profile/Tim.jpeg';
+import Phil from '../assets/profile/Phil.jpeg';
+import Ramona from '../assets/profile/Ramona.jpeg';
+import Akin from '../assets/profile/Akin.jpeg';
+import Martin from '../assets/profile/Martin.jpeg';
+import Tina from '../assets/profile/Tina.jpeg';
+import Leonie from '../assets/profile/Leonie.jpeg';
+import Park from '../assets/hobbies/park.jpeg';
+import Climbing from '../assets/hobbies/climbing.jpeg';
+import YogaImage from '../assets/hobbies/Yoga.jpeg';
+import Dom from '../assets/hobbies/dom.jpeg';
+import Arts from '../assets/hobbies/arts.jpeg';
+import CookingMenu from '../assets/hobbies/cooking.jpg';
+import Skate from '../assets/hobbies/skate.jpeg';
+import Poetry from '../assets/hobbies/poetry.jpeg';
+import Party from '../assets/hobbies/party.jpeg';
```

Comment:
All these imports could be better organized with a `index.js`.
See https://javascript.info/import-export#re-export

Imagine this:
```
import { Phil, Ramona, Akin, Martin, Tina, Leonie }from '../assets/profile';
```

I am aware that these might be test profiles, but it's still something you could improve here.

Reply:
good idea, I will do it in the refactoring round :)